src/java_bytecode/java_entry_point.cpp

```diff
@@ -46,32 +47,33 @@ Function: create_initialize
 
 \*******************************************************************/
 
-namespace {
```

Comment:
yes, this should be in a class. you can make it static as an intermediate step before someone refactors it